images/runtime/node/12.5/Dockerfile.base

```diff
@@ -0,0 +1,88 @@
+FROM oryx-run-base AS node-original
```

Comment:
Let's just add the file under "12", since this will be a major-version only option